dotcom-rendering/src/web/components/FetchCommentCounts.importable.tsx

```diff
@@ -1,6 +1,8 @@
+import { CacheProvider } from '@emotion/react';
 import { useEffect, useState } from 'react';
-import { renderToString } from 'react-dom/server';
+import ReactDOM from 'react-dom';
```

Comment:
I think you could import the render method directly

```suggestion
import { render } from 'react-dom/server';
```